docs/manual/en/manual.txt

```diff
@@ -0,0 +1,256 @@
+. alternating sine
+. camel sine, also known as absolute alternating sine
+. square
+. logarithmic sawtooth, also known as derived square
+
+==== Envelopes
+
+The envelope generator is controlled by four dials labelled "ADSR".
+The envelope controls the amplitude (volume) of an oscillator with respect to time.
+
+.ADSR Envelope
+image::../resources/adsr.svg[250,250]
+
+The attack (A) and decay (D) parameters control the durations of those envelope stages.
+
+The sustain (S) parameter controls the volume level held after A + D has elapsed, and as long as the key remains pressed.
+However, if the "Sustain" flag is disabled, there is no sustain stage, and the envelope effectively becomes an "ADR".
+
+The release (R) parameter comes after sustain, and similarly to "A" and "D", specifies the duration of the release stage.
+
+==== Flags
+
+The "Tremolo" and "Vibrato" flags apply small modulations to the operator, affecting volume and frequency respectively.
+
+"Sustain" enables or disables the "S" stage of the envelope. Refer to the explanation on envelopes.
+
+"Key scaling" automatically shortens the duration of the envelope as the pitch of the note increases.
+
+==== Levels
+
+The output level "Lv" is the global volume of the operator.
+Note that there is no "0" setting; even at the lowest level, the operator can be heard faintly.
+
+The frequency multiplier "F*" controls the base frequency of the operator, in octaves.
+
+The key scale level "Ksl" determines the extent to which amplitude is attenuated as pitch increases.
+
+=== Other parameters
+
+The remaining parameters fall into two groups: Miscellaneous and Global.
+
+A miscellaneous parameter applies to the currently selected program in the editor. On the other hand, a global parameter is a property of the bank, and applies to all programs in that bank.
+
+.Miscellaneous parameters
+image::../resources/misc.png[200,200]
+
+"Velocity Offset" is a fixed modifier applied to the velocity of MIDI notes, and can be set in the range [0, 127]. A positive offset increases velocity, and a negative value decreases it.
+
+.Global parameters
+image::../resources/global.png[230,230]
+
+The "Deep tremolo" and "Deep vibrato" flags make those effects more pronounced, if they are active. The effect depths are fixed values determined by the design of the YMF262.
+
+"Volume moodel" specifies a function for transforming the discrete parameters associated with volume into actual gain values. In most cases, the generic model is appropriate.
+
+However, when loading banks from other software, the formula that reproduces the operation of that software should be selected. For example, if you load a bank from Doom, you should choose the DMX model. For Duke Nukem 3D, use Apogee.
+
+== About
+
+ADLplug was developed by Jean Pierre Cimalando in 2018. +
+It is based on previous work by several groups of people.
+
+The YMF262 emulators were developed by the DOSBox team, and by Alexey Khokholov (Nuke.YKT) for Nuked OPL.
```

Comment:
May I add credits for the other emulator cores included?

Reply:
Yes the documentation was not updated to reflect these additions.

On a side note, there are also more credits I wish to add of non-developers, (also to add in the software itself).

**Intrument design**
- Sneakernets
- Papiezak
- Vitaly Novichkov

**Artist**
- Brian (linuxsynths.com)
- Garvalf

**Translator**
- Olivier Humbert
- Bruce Sutherland

**Artwork**
- Christopher Arndt